src/GameOptions/ui/KeyBindingPage.tsx

```diff
@@ -259,45 +283,65 @@ export function KeyBindingPage(): React.ReactElement {
     );
   };
 
+  const keyBindingRows = getRecordKeys(CurrentKeyBindings)
+    .filter(
+      (keyBindingType) =>
+        knowAboutBitverse() || !(SpoilerKeyBindingTypes as unknown as string[]).includes(keyBindingType),
+    )
+    .map((keyBindingType) => {
+      const primaryKeyCombination = CurrentKeyBindings[keyBindingType][0] ? (
```

Comment:
Optional: Would this be cleaner as a loop from 0 to 2? It *looks* like everything is just duplicated, but I might have missed something that's subtly different between them.